django_fsm_freeze/models.py

```diff
@@ -16,10 +16,12 @@
 
 
 @contextmanager
-def bypass_fsm_freeze(objs: Union[object, Iterable]):
+def bypass_fsm_freeze(
+    objs: Union['FreezableFSMModelMixin', Iterable['FreezableFSMModelMixin']]
+):
     if not isinstance(objs, Iterable):
         objs = (objs,)
-    errors = list()
+    errors = []
```

Comment:
This is good 👍  since this PR is closed - I will include this in another improvement PR #17